spacy_llm/models/rest/cohere/registry.py

```diff
@@ -1,31 +1,26 @@
-from typing import Any, Callable, Dict, Iterable, Type
-
+from typing import Any, Callable, Dict, Iterable
 
 from spacy.util import SimpleFrozenDict
 
-from . import anthropic, base, openai, noop
-from ...registry import registry
-
+from ....compat import Literal
+from ....registry import registry
+from .model import Cohere, Endpoints
 
-supported_apis: Dict[str, Type[base.Backend]] = {
-    "OpenAI": openai.OpenAIBackend,
-    "Anthropic": anthropic.AnthropicBackend,
-    "NoOp": noop.NoOpBackend,
-}
 
-
-@registry.llm_backends("spacy.REST.v1")
-def backend_rest(
-    api: str,
+@registry.llm_models("spacy.command.v1")
+def cohere_command(
```

Comment:
This is a case that I hadn't considered, that "command" is kind of a pretty generic name, though I guess everyone knows this is from Cohere? 😕

Reply:
It's a weird name, yes. We should document this explicitly (see [here](https://github.com/explosion/spacy-llm/pull/176#issuecomment-1611662901)).